arangod/Pregel/GraphStore.cpp

```diff
@@ -515,12 +520,12 @@ void GraphStore<V, E>::loadEdges(
   size_t addedEdges = 0;
   auto buildEdge = [&](Edge<E>* edge, std::string_view toValue) {
     ++addedEdges;
-    ++_observables.edgesLoaded;
+    ++(*_observables.edgesLoaded);
```

Comment:
This should be done only when we are sure that the edge is really added and no error occurs.